test/DynamoCoreWpfTests/PythonNodeCustomizationTests.cs

```diff
@@ -270,6 +270,155 @@ public void OnMoreInfoEventTest()
+                                .FirstOrDefault();
+
+            //Validate tha the documentation browser tab is opened and visible once we clicked the button.
+            Assert.IsNotNull(learnMoreTab);
+            Assert.IsTrue(learnMoreTab.IsVisible);
+        }
+
+        /// <summary>
+        /// PythonStringNode
+        /// This test method will click in the "Learn more about Python" button and the OpenPythonLearningMaterial method from PythonStringNode.cs will be executed
+        /// </summary>
+        [Test]
+        public void OpenPythonLearningMaterial_PythonNodeFromStringValidationTest()
+        {
+            Open(@"core\python\pythonFromString.dyn");
+
+            var nodeView = NodeViewWithGuid("bad59bc8-9b49-47b6-99ee-34fa8dca91ae");
+            var nodeModel = nodeView.ViewModel.NodeModel as PythonNodeBase;
+            Assert.NotNull(nodeModel);
+
+            var learnMoreMenuItem = nodeView.MainContextMenu
+                .Items
+                .Cast<MenuItem>()
+                .First(x => x.Header.ToString() == "Learn more about Python");
+
+            Assert.IsNotNull(learnMoreMenuItem);
+
+            //Click the button and internally the  OpenPythonLearningMaterial method is executed
+            learnMoreMenuItem.RaiseEvent(new RoutedEventArgs(MenuItem.ClickEvent));
+
+            DispatcherUtil.DoEvents();
+
+            var learnMoreTab = this.View.ExtensionTabItems
+                                .Where(x => x.Content.GetType().Equals(typeof(DocumentationBrowserView)))
+                                .FirstOrDefault();
+
+            //Validate tha the documentation browser tab is opened and visible once we clicked the button.
+            Assert.IsNotNull(learnMoreTab);
+            Assert.IsTrue(learnMoreTab.IsVisible);
+        }
+
+        /// <summary>
+        /// PythonStringNode
+        /// This test method will click in the "Python Engine Version" menu option and the UpdateToPython3Engine() method from PythonStringNode.cs will be executed
+        /// </summary>
+        [Test]
+        public void UpdateToPython3Engine_PythonStringNodeTest()
+        {
+            Open(@"core\python\pythonFromString2to3Test.dyn");
+
+            var nodeView = NodeViewWithGuid("bad59bc8-9b49-47b6-99ee-34fa8dca91ae");
+            var nodeModel = nodeView.ViewModel.NodeModel as PythonNodeBase;
+
+            //Get the Watch node so we can check the content later
+            var watchOUT = Model.CurrentWorkspace.NodeFromWorkspace<Watch>("971220846c1b4d54aa11cca81f417c2b");
+            Assert.NotNull(nodeModel);
+
+            //Get the python engine menu 
+            var engineMenuItem = nodeView.MainContextMenu
+                .Items
+                .Cast<MenuItem>()
+                .First(x => x.Header.ToString() == "Python Engine Version");
+
+            Assert.IsNotNull(engineMenuItem);
+
+            //Get the python engime menu CPython3
```

Comment:
typo `engine`

Reply:
@QilongTang fixed in the commit.
https://github.com/DynamoDS/Dynamo/pull/11156/commits/0036fd5fff2814eb944d45af4885f9242fd444e8